torch/testing/_internal/common_device_type.py

```diff
@@ -796,6 +813,35 @@ def __call__(self, fn):
         fn.precision_overrides = self.d
         return fn
 
+# Specifies per-dtype tolerance (rtol, atol) overrides. It has priority over
+# precisionOverride.
+# Ex.
+#
+# @toleranceOverride({torch.float : (1e-3, 1e-2}, torch.double : {0, 1e-4}, })
```

Comment:
`toleranceOverride({torch.float : (1e-3, 1e-2), torch.double : (0, 1e-4) })`

I think it would be nice if it is a named tuple or somewhere we can visually see `1e-3` is rtol and `1e-2` is atol.

cc: @mruberry  what do you think?

Reply:
How about calling 'tol' for the namedtuple and doing something like this:
```
tol = namedtuple('tol', ['rtol', 'atol'], defaults=[0.0, 0.0])
decorators=(toleranceOverride({torch.float32: tol(rtol=3.9e-6), }),),
```

Also do you prefer 'rtol' or 'atol' as the first item? For users probably 'atol' is the default choice, but the dtype_precisions in common_utils.py uses (rtol, atol).